src/common/blockchain.sh

```diff
@@ -1,3 +1,201 @@
 #!/usr/bin/env bash
 
 # Common IBM blockchain platform functions, e.g. to provision a blockchain service
+
+# shellcheck disable=2086
+
+# shellcheck source=src/common/utils.sh
+source "${SCRIPT_DIR}/common/utils.sh"
+
+function setup_service_constants {
```

Comment:
Can we document what each of these functions are supposed to do? Just a few comments on top of each function so that it is understood that the module is for.  It should also include a list of the arguments the function is expecting in the order it is expected and a description.

Reply:
https://github.com/IBM-Blockchain-Starter-Kit/build-lib/pull/40/commits/df494154dd51497845e3e1b75e0cc529cd79c0fa